alibi_detect/cd/mmd.py

```diff
@@ -82,27 +89,35 @@ def __init__(
         backend = backend.lower()
         BackendValidator(
             backend_options={'tensorflow': ['tensorflow'],
-                             'pytorch': ['pytorch']},
+                             'pytorch': ['pytorch'],
+                             'keops': ['keops']},
             construct_name=self.__class__.__name__
         ).verify_backend(backend)
 
         kwargs = locals()
         args = [kwargs['x_ref']]
         pop_kwargs = ['self', 'x_ref', 'backend', '__class__']
+        if backend == 'tensorflow' and has_tensorflow:
+            pop_kwargs += ['device', 'batch_size_permutations']
+            detector = MMDDriftTF
+        elif backend == 'pytorch' and has_pytorch:
+            pop_kwargs += ['batch_size_permutations']
+            detector = MMDDriftTorch
+        else:
```

Comment:
The `else` here is not consistent with the previous `if backend == 'tensorflow' and has_tensorflow:` and `elif backend == 'pytorch' and has_pytorch:`, since it does not check `has_keops` like the other two. That being said, I don't believe we need any `has_<backend>` checks here (See https://github.com/SeldonIO/alibi-detect/issues/576).